zeppelin/hdi_add_zeppelin.sh

```
@@ -0,0 +1,220 @@
+#! /bin/bash
+
+#----------------------------------------------------------------
+# THIS SCRIPT ADDS INSTALLS AND STARTS ZEPPELIN ON HBASE CLUSTER 
+#----------------------------------------------------------------
+
+
+#----------------------------------------------------------------
+# PRINT USAGE INFORMATION
+#----------------------------------------------------------------
+
+
+print_usage()
```

what are the cluster versions that we are going to support? Is there a way to fail fast in case the cluster version is not supported?

---

we will support only 2.6